Source/WebCore/Modules/applicationmanifest/ApplicationManifest.h

```diff
@@ -36,6 +36,12 @@
 namespace WebCore {
 
 struct ApplicationManifest {
+    enum class Direction : uint8_t {
+        Auto,
+        Ltr,
+        Rtl,
```

Comment:
Shouldn't the casing be LTR and RTL?

Reply:
That would be great, but then the linter complains and wants to have either `kLTR` or `Ltr`.

Reply:
It's okay to ignore the linter when it is wrong, although https://webkit.org/code-style-guidelines/#names-basic does not making a ruling either way. As I'd like someone else to review anyway, hopefully they can look at this as well.

Reply:
I flagged the same thing... we might just need to disable the linter for this. That's ok.

Reply:
To be more precise, we often code exceptions into the linter for cases like these, rather than "disabling" it; I think it’s pretty easy to add one and that change can be included in the pull request. By the way, for such a short list I often suggest using a single-line format, but I guess that’s not happening below for `Display`.

Reply:
Yes, sorry for not being clear. By "disabling it" I meant a one line exception.

Reply:
For reference if anyone lands here:  `git grep "// NOLINT"`